classes/store.php

```diff
@@ -88,7 +88,7 @@ public function get($type, $key) {
      * @param mixed    $value  The value.
      * @param int|null $expire The expiration time (unix timestamp), or NULL if it never expires.
      */
-    public function set($type, $key, $value, $expire = null) {
+    public function set(string $type, string $key, $value, ?int $expire = null): void {
         global $DB;
 
         assert(is_string($type));
```

Comment:
I dont think this assertion is required at all now that we are type guarded in the parameters

Reply:
I kept in one of the assertions but changed it slightly as it's checking for a particular value not covered by the param types.